### PR TITLE
fix: improve actisense error message when serialport is missing

### PR DESCRIPTION
**Pull request by sbender9**

### CODE REVIEW

File: packages/streams/simple.js [hunk 1]
```diff
@@ -187,8 +187,12 @@ const pipeStartByType = {
 
 function nmea2000input (subOptions, logging) {
   if (subOptions.type === 'ngt-1-canboatjs') {
+    const actisenseSerial = require('./actisense-serial')
```
Comment (tkurki) [inline packages/streams/simple.js:190]: Doesn't the `require` call throw when it fails because of a missing `serialport`?

Comment (sbender9) [reply packages/streams/simple.js:190]: No. There's a try/catch so that the rest of canboatjs can be used when serialport is missing.

Comment (tkurki) [reply packages/streams/simple.js:190]: ok. Lint will barf on the next line though?

Comment (sbender9) [reply packages/streams/simple.js:190]: the was the cause of the `(intermediate value) is not a function` error message reported the other day

**Review by tkurki — APPROVED** [review]

**Pull request merged**